main/lsp/test/lsp_preprocessor_test.cc

```diff
@@ -149,7 +149,7 @@ TEST_CASE("IgnoresWatchmanUpdatesFromOpenFiles") {
     CHECK_EQ(updates->updates[0]->source(), fileContents);
 }
 
-// When deepCopying initialGS for typechecking, it should always have all previous updates applied to it.
+// it.
```

Comment:
```suggestion
```

Reply:
I did a good job rephrasing that and then only removing half of it. Thank you for catching this :)